.github/workflows/build.yml

```diff
@@ -51,6 +51,7 @@ jobs:
         run: |
           cd vendor/cakephp/app
           composer install --prefer-dist --no-progress --no-suggest
+          cp ../../../tests/app.php ./config/app.php
```

Comment:
It is not recommended to modify the default configuration in this way, you can modify it at the code level .